WooCommerce/Classes/POS/Presentation/Order Messages/PointOfSaleOrderSyncCouponsErrorMessageView.swift

```diff
@@ -126,3 +123,17 @@ private extension String {
         return AttributedString(self)
     }
 }
+
+#Preview {
+    if #available(iOS 17.0, *) {
+        PointOfSaleOrderSyncCouponsErrorMessageView(message: "An error happened!") {}
+            .environment(POSPreviewHelpers.makePreviewAggregateModel())
+    }
+}
+
+#Preview {
+    if #available(iOS 17.0, *) {
+        PointOfSaleOrderSyncCouponsErrorMessageView(message: "Lo siento, este cupón no se puede aplicar a los productos seleccionados.") {}
```

Comment:
Nit: The error message that comes from the backend is translated incorrectly so you got tricked! 😂, should be something like _"Lo sentimos, éste cupón no se puede aplicar a los productos seleccionados."_ I'll check out where to report this

Reply:
Thanks for taking care of the translation quality as well!